lucien-ios/AppDelegate.swift

```diff
@@ -17,7 +17,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         let window = UIWindow(frame: UIScreen.main.bounds)
         self.window = window
-        window.rootViewController = RootViewController()
+        window.rootViewController = StartMyCollectionViewController()
```

Comment:
Is this change how the actual app works or is it a temp change for your testing? If it's a temp change it shouldn't be in the PR

Reply:
This was a temporary change for testing. I forgot to change it.